Marlin/src/gcode/feature/power_monitor/M430.cpp

```diff
@@ -40,13 +40,13 @@ void GcodeSuite::M430() {
   bool do_report = true;
   #if HAS_WIRED_LCD
     #if ENABLED(POWER_MONITOR_CURRENT)
-      if (parser.seen('I')) { power_monitor.set_current_display(parser.value_bool()); do_report = false; }
+      if (parser.seenval('I')) { power_monitor.set_current_display(parser.value_bool()); do_report = false; }
```

Comment:
`seenval` is inappropriate when used with `value_bool`. The parameter by itself counts as `1` / `true`.